src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -408,20 +410,23 @@ def get_analyze_ctx(self, branch_name: str, git_obj: Git) -> AnalyzeContext:
 
         Parameters
         ----------
-        git_obj : Git
-            The pydriller Git object of the target repository.
         branch_name : str
             The name of the branch that we are analyzing.
             We need this because when the target repository is in a detached state,
             the current branch name cannot be determined.
+        git_obj : Git
+            The pydriller Git object of the target repository.
 
         Returns
         -------
         AnalyzeContext
             The context of object of the target repository.
         """
-        remote_path = git_url.get_remote_url_of_local_repo(git_obj)
-        full_name = git_url.get_repo_full_name_from_url(remote_path)
+        origin_remote_path = git_url.get_remote_origin_of_local_repo(git_obj)
```

Comment:
Please add comments to describe the approach here.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/153/commits/bdeba388db589ff62f060dbb87c5512bb05ba587